### PR TITLE
add vitest to readme and update cargo-nextest

### PR DESCRIPTION
**Pull request by joshmarinacci**
Also update some spelling.

### CODE REVIEW

File: code-quality/linters/supported/buildifier.md [hunk 1]
```diff
@@ -41,6 +41,6 @@ You can move these files to `.trunk/configs` and `trunk check` will still find t
 
 ## Links
 
-- [buildifier site](https://github.com/bazelbuild/buildtools/blob/master/buildifier/README.md)
-- buildifier Trunk Code Quality [integration source](https://github.com/trunk-io/plugins/tree/main/linters/buildifier)
+- [Buildifier  site](https://github.com/rhysd/actionlint)
```
Comment (gewenyu99) [inline code-quality/linters/supported/buildifier.md:44]:
```suggestion
- [Buildifier site](https://github.com/rhysd/actionlint)
```

There's an extra space here.

File: code-quality/linters/supported/clang-format.md [hunk 2]
```diff
@@ -16,51 +14,32 @@ layout:
...
 ## Settings
 
-ClangFormat supports the following config files:
+clang-format supports the following config files:
 * `.clang-format`
 
 You can move these files to `.trunk/configs` and `trunk check` will still find them. See [Moving Linter Configs](..#moving-linter-configs) for more info.
 
 
-## Usage Notes
```
Comment (gewenyu99) [inline code-quality/linters/supported/clang-format.md:41]: intentional or smthing wrong with script?

File: code-quality/linters/supported/gofmt.md [hunk 3]
```diff
@@ -14,26 +20,26 @@ layout:
...
+You can enable the Gofmt  formatter with:
 
 ```shell
 trunk check enable gofmt
 ```
 
 ## Auto Enabling
 
-gofmt will be auto-enabled if any *Go* files are present.
+Gofmt  will be auto-enabled if any *Go* files are present.
```
Comment (gewenyu99) [inline code-quality/linters/supported/gofmt.md:35]:
```suggestion
Gofmt will be auto-enabled if any *Go* files are present.
```

extra space

File: code-quality/linters/supported/tflint.md [hunk 4]
```diff
@@ -14,23 +14,23 @@ layout:
     visible: true
 ---
 
-# tflint
+# TFLint
```
Comment (gewenyu99) [inline code-quality/linters/supported/tflint.md:17]: Extra space

File: flaky-tests/frameworks/vitest.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    visible: false
+  tableOfContents:
+    visible: true
+  outline:
+    visible: true
+  pagination:
+    visible: true
+---
+
+Vitest is a testing framework for Javascript.
```
Comment (gewenyu99) [inline flaky-tests/frameworks/vitest.md:17]:
```suggestion
Vitest is a testing framework for JavaScript.
```

File: flaky-tests/frameworks/vitest.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+      ['junit', { suiteName: 'custom suite name', classname: 'custom-classname' }]
+    ]
+  },
+})
+```
+For more details see the [Vitest JUnit Reporter docs](https://vitest.dev/guide/reporters#junit-reporter).
+
+
+
+
```
Comment (gewenyu99) [inline flaky-tests/frameworks/vitest.md:53]:
```suggestion
We should really get rid of these trailing spaces from the script 📦 
```

**Review by gewenyu99 — CHANGES_REQUESTED** [review]

File: code-quality/linters/supported/buildifier.md (hunk 1 above)
Comment (joshmarinacci) [reply code-quality/linters/supported/buildifier.md]: extra space in the Dato entry. Fixed.

**Pull request merged**